staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -82,6 +84,13 @@ type objState struct {
 	stale bool
 }
 
+type transformedObj struct {
```

Comment:
`Obj` seems like the wrong suffix.
How about `transformedKV`?

Reply:
Renamed to transformedKVPairs.